src/collective_modes.jl

```diff
@@ -0,0 +1,66 @@
+module collective_modes
+
+# For a particularly elegant derivation of collective frequency shifts and decay rates in 1D
+# atomic chains, see Asenjo-Garcia et al 10.1103/PhysRevX.7.031024.
+
+import LinearAlgebra
+import Polylogarithms
+
+pl = Polylogarithms
+
+"""
+    Omega_k_chain(k, a, polarization)
+
+Collective frequency shift Omega_k of mode k for an infinite chain of atoms along x-axis.
+
+WLOG, this calculation scales natural atomic linewidth lambda0=1 and decay rate gamma0=1.
```

Comment:
I'm a bit confused with the terminology. To me, the "natural atomic linewidth" is the decay rate. Are you referring to the transition wavelength as `lambda0`?